modules/bridge/src/main/ts/ephox/bridge/core/ComponentSchema.ts

```diff
@@ -24,7 +24,10 @@ export const optionalSelect = FieldSchema.optionFunction('select');
 
 export const active = FieldSchema.defaultedBoolean('active', false);
 export const borderless = FieldSchema.defaultedBoolean('borderless', false);
+// TODO: replace `disabled` with `enabled` field
```

Comment:
What does this mean exactly should this be a follow up ticket. Is the disabled working as well as the enabled or is it just there to satisfy the field validation?

Reply:
I wasn't sure where else requires `disabled` at the time. but can confirm that all tests pass after removing it